webrtc.html

```diff
@@ -4946,6 +4947,23 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <code>degradationPreference</code> indicates which is
               preferred.</p>
             </dd>
+            <dt><dfn><code>upscaleAllowed</code></dfn> of type
+            <span class="idlMemberType"><a>boolean</a></span>,
+            defaulting to <code>false</code></dt>
+           <dd>
+             <p>When, and only when, this value is set true the browser MAY
+            upscale video temporally or spatially, or upscale the audio sampling
+            rate, or upscale the number of bits in the audio sample that this
+            <code><a>RtpSender</a></code> sends over RTP.
+            Things like image size can be negotiated to be at lease a minimum
+            size.  However, the sources camera may not be capable of producing
+            an image of that size.
+            If this value is set to true, the browser MAY upscale the 
+            video to produce a larger image. If the value was set to false, or the
+            browser did not implement up scaling, then sending RTP would need to
+            cease and an error generated since since the negotiated frame rate
```

Comment:
we do not have any error for this(?). It feels like this is a similar situation to circuit breakers stopping the flow.